src/client/components/Modal.tsx

```diff
@@ -0,0 +1,92 @@
+import React, { useEffect, useRef, useState } from 'react'
+import ReactDOM from 'react-dom'
+import styled from "styled-components";
+
+const Modal = ({ show, onClose }) => {
+  const [isBrowser, setIsBrowser] = useState(false)
+
+  useEffect(() => {
+    setIsBrowser(true)
+  }, [])
+
+  const handleCloseClick = (e) => {
+    e.preventDefault();
+    onClose();
+  };
+
+
+  const modalcontent = show ? (
+    <StyledModalOverlay>
+      <div className="modal-overlay" />
+      <StyledModal>
+        <StyledModalHeader>
+          {/* <div
+        className="modal-wrapper"
+        aria-modal
+        aria-hidden
+        tabIndex={-1}
+        role="dialog"
+      >
+        <div className="modal"> */}
```

Comment:
No commented code

Reply:
Yeah, I will make the corrections! Thank you